proposals/lip-0050.md

```diff
@@ -139,25 +150,61 @@ accountReclaimedEventDataSchema = {
     "properties": {
         "legacyAddress": {
             "dataType": "bytes",
-            "length": 8,
+            "length": NUM_BYTES_LEGACY_ADDRESS,
             "fieldNumber": 1
         },
         "address": {
             "dataType": "bytes",
-            "length": 20,
+            "length": NUM_BYTES_ADDRESS,
             "fieldNumber": 2
         },
         "amount": {
             "dataType": "uint64",
             "fieldNumber": 3
+        }
+    }
+}
+```
+
+#### ReclaimFailed
```

Comment:
After reading the command logic below, it seems that this event is only emitted during verification. It could then be removed (as we are not supposed to emit during verification).